src/pivot.core.coffee

```diff
@@ -18,10 +18,10 @@ numberFormat = (opts) ->
     digitsAfterDecimal: 2, scaler: 1,
     thousandsSep: ",", decimalSep: "."
     prefix: "", suffix: ""
-    showZero: false
+    showZero: true
   opts = $.extend defaults, opts
   (x) ->
-    return "" if isNaN(x) or not isFinite(x)
+    return "" if isNaN(x) or (x is null) or not isFinite(x)
```

Comment:
what about

    return "" if isNaN(x) or x? or not isFinite(x)

or do we only truly care if `x` is `null` and not `undefined`?